h/renderers.py

```diff
@@ -0,0 +1,31 @@
+# Taken from https://pyramid-cookbook.readthedocs.org/en/latest/templates/customrenderers.html
+# with minor modifications
+import csv
+try:
+    import StringIO as io # python 2
```

Comment:
Two minor comments here. 

1. We try to centralise all the Py2/Py3 switching we do in one place, namely `h/_compat.py`, so this should move there.
2. This actually imports different tokens in Python 2 and Python 3 (`io` in Python 2 and `StringIO` in Python 3.)

So I'd suggest something like this in `h/_compat.py`:

```python
try:
    from io import StringIO
except ImportError:
    from StringIO import StringIO
```

which you can then use as:

```python
from h._compat import StringIO
```